source/scripts/modules/serlo_injections.js

```diff
@@ -0,0 +1,166 @@
+/**
+ *
+ * Athene2 - Advanced Learning Resources Manager
+ *
+ * @author  Julian Kempff (julian.kempff@serlo.org)
+ * @license LGPL-3.0
+ * @license http://opensource.org/licenses/LGPL-3.0 The GNU Lesser General Public License, version 3.0
+ * @link        https://github.com/serlo-org/athene2 for the canonical source repository
+ */
+
+/*global define, require, window, Modernizr*/
+require(['http://www.geogebratube.org/scripts/deployggb.js']);
+
+define(['jquery', 'common', 'translator', 'content'], function ($, Common, t) {
+    "use strict";
+    var Injections,
+        cache = {},
+        ggbApplets = {},
+        ggbAppletsCount = 0,
+        geogebraScriptSource = 'https://www.geogebra.org/web/4.4/web/web.nocache.js',
+        $geogebraTemplate = $('<article class="geogebraweb" data-param-width="700" data-param-height="525" data-param-usebrowserforjs="true" data-param-enableRightClick="false"></article>'),
+        //gtApplets = {},
+        //geogebraTubeScriptSource = 'http://www.geogebratube.org/scripts/deployggb.js',
+        gtAppletsCount = 0,
+        $geogebraTubeTemplate = $('<embed style="width:600px; height:400px" />');
```

Comment:
Shouldn't this be `$geogebraTubeTemplate = $('<div style="width:100% overflow:hidden"></div>');`?